docs/source/advanced/mapping.rst

```diff
@@ -1,5 +1,24 @@
+Mapping onto Specific Hardware Architectures
+============================================
+
+By providing an accelerator independent interface for kernels, their execution and memory accesses at different hierarchy levels, *alpaka* allows the user to write accelerator independent code that does not neglect performance.
+
+The mapping of the decomposition to the execution environment is handled by the back-ends provided by the *alpaka* library as well as user defined back-ends.
```

Comment:
I think we need to be consistent with spelling: alway `backend` or always `back-end`. I am for the former.

Reply:
Yes, good point. In general, I use `backend`. The version with `-` was written by another person. So I prefer also the former version.

Reply:
@sbastrakov 
I greped over the whole project. Most of the time, `back-end` is used. One big exception are the unified HIP/CUDA files, which use `backend`. I would say, we use `back-end`, because it is mostly used in the project, in special in messages in cmake.

Reply:
Okay. I think all 3 are not bad, and consistency is most important.